packages/salesforcedx-utils-vscode/src/config/configUtil.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { ConfigAggregator, ConfigFile, ConfigValue } from '@salesforce/core';
+import * as path from 'path';
+import { TelemetryService } from '../telemetry';
+import { getRootWorkspacePath } from '../workspaces';
+
+export enum ConfigSource {
+  Local,
+  Global,
+  None
+}
+
+// This class should be reworked or removed once the ConfigAggregator correctly checks
+// local as well as global configs. It's also worth noting that ConfigAggregator, according
+// to its docs checks local, global and environment and, for our purposes, environment may
+// not be viable.
+
+export class ConfigUtil {
```

Comment:
Minimal config function necessary to maintain current functioning of workspace context.